docs/pipelineruns.md

```diff
@@ -279,6 +279,7 @@ spec:
   podTemplate:
     securityContext:
       runAsNonRoot: true
+      runAsUser: 1001
```

Comment:
Maybe update the last sentence of the paragraph describing this example from `and executes it as a non-root user.` to `and executes it as user 1001.`?

Reply:
Added